ctapipe/instrument/tests/test_camera.py

```diff
@@ -87,7 +91,10 @@ def test_neighbor_pixels(cam_id):
         assert n_neighbors.count(5) == 0
         assert n_neighbors.count(6) == 0
 
-    assert n_neighbors.count(1) == 0  # no pixel should have a single neighbor
+    # whipple has inhomogenious pixels that mess with pixel neighborhood
+    # calculation
+    if cam_id != 'Whipple490':
```

Comment:
here can use an `AssertRaises` if you implement the exception as described above.  Then you can test if a slice of the camera still works (e.g. `geom[0:378]`)

Reply:
The neighbors it calculates are not completly wrong. Just the constrained that no pixel should have just one neighbour is not met. So Maybe don't throw an exception.

Reply:
See here:

![whipple](https://user-images.githubusercontent.com/5488440/54550251-f4fd7f80-49ab-11e9-8f7b-8523f3a32fea.gif)

Reply:
It works better than I thought. I never really checked with that camera.  I guess we can just keep it that way then until it becomes a real problem. Probably this will rarely if never be an issue anyhow